manage_breast_screening/data/demo_data.yml

```diff
@@ -0,0 +1,562 @@
+providers:
+  - name: "West Tester Hospital Trust"
+    id: 2a3e50e9-7e49-4c2b-8c9c-9b1f7b3a1aa4
+    settings:
+      - name: "West Tester BSU"
+        id: 8f2a1b3c-4d5e-6f7a-8b9c-0d1e2f3a4b5c 
```

Comment:
Another idea which may not work - perhaps we could avoid specifying IDs everywhere and instead generate fake UUIDs based on a hash of natural keys like nhs number. Might be more trouble than it's worth though, just coming at this from the angle of making this file as readable as possible for non-devs